scripts/snap/snapcraft.template.yaml

```diff
@@ -50,4 +50,4 @@ parts:
       cp -v ethkey $SNAPCRAFT_PART_INSTALL/usr/bin/ethkey
       cp -v ethstore $SNAPCRAFT_PART_INSTALL/usr/bin/ethstore
       cp -v whisper $SNAPCRAFT_PART_INSTALL/usr/bin/whisper
-    stage-packages: [libc6, libudev1, libstdc++6, cmake, libdb5.3]
```

Comment:
we added libc6 because it was failling without. It's working now?

Reply:
and it was the reason it failed here: https://github.com/paritytech/devops/issues/307

Reply:
yes